specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -904,6 +1058,45 @@
         }
       }
     },
+    "IscsiTargetUpdateProperties": {
+      "type": "object",
+      "description": "Properties of iSCSI Target request payload.",
+      "required": [
+        "tpgs"
+      ],
+      "properties": {
+        "provisioningState": {
+          "description": "State of the operation on the resource",
+          "type": "string",
+          "$ref": "#/definitions/ProvisioningState",
+          "readOnly": true
+        },
+        "status": {
+          "type": "string",
+          "enum": [
+            "Invalid",
+            "Unknown",
+            "Healthy",
+            "Unhealthy"
+          ],
+          "x-ms-enum": {
+            "name": "OperationalStatus",
+            "modelAsString": true
+          },
+          "readOnly": true,
+          "description": "Operational status of the Disk pool"
+        },
+        "tpgs": {
+          "type": "array",
+          "description": "list of iSCSI target portal groups",
+          "items": {
+            "$ref": "#/definitions/TargetPortalGroup"
+          },
+          "maxLength": 1,
+          "minLength": 1
```

Comment:
As there must be 1 tpg, why define as array?

Reply:
Right now the RP supports only 1 value -- but in the future we would be able to support more